Gemfile.lock

```diff
@@ -196,7 +196,6 @@ GEM
     websocket-driver (0.6.5)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.2)
-    xmlrpc (0.3.0)
```

Comment:
You have to be careful when committing the lock file. This dep installed only on R2.4+ and we need to keep it locked. So either use R2.4+ in your devel env or remember to skip `Gemfile.lock` on commits not dealing specifically with dependency updates.